documentation/reference/qfieldcloud/projects.en.md

```diff
@@ -34,9 +49,21 @@ project
 │   ├── bees-20220404121212.jpg
 │   ├── bees-20220405040506.jpg
 │   └── fields-20220405040607.jpg
-└── project.qgs
+├── project.qgs
+├── project_attachments.zip
+└── project.qml
 ```
 
+
+The file in a QGIS project can be in one of the following groups by their purpose:
```

Comment:
The files in a QFieldCloud project can be in one of the following groups by their purpose: